### DISCUSSION TITLE
请教：大家是怎么设置国内DNS的？

### DISCUSSION THREAD

**Discussion by Platway** [2dust/v2rayNG#2981] [General]
照理说，最快最稳定的应该是运营商的DNS，但是在v2rayNG里不能设置运营商DNS，因为运营商可能不停切换，比如有时连移动网络，有时又可能连接不同的WiFi。如果设置成默认的223.5.5.5，或者腾讯的119.29.29.29等等，时常不稳定，有时会被劫持。想问问大家是怎么比较好地处理这个问题的？

**Comment by Fubuki0x10DE** [comment]
用DNS over HTTPS，Microsoft Edge就支援。

**Comment by laaArmy** [comment]
local？
